controls/pcidss_4_ocp4.yml

```diff
@@ -2845,10 +2856,9 @@ controls:
           name and protocol).
       levels:
         - base
-      status: pending
-      notes: |-
-        Standard settings for audit should be enough.
-      rules: []
+      status: inherently met
+      notes: >-
+        The logs generated by OpenShift and Red Hat CoreOS include all the data required.
```

Comment:
Do we get all this with the `Default` audit profile? Do we need to include `audit_profile_set` as part of this control?

Reply:
Hmm, the PCI-DSS v3.2.1 goes with the `Default`audit profile.

But based on:
```
10.2.1.7 Audit logs capture all creation and deletion
of system-level objects.
```
and the **what** listed in:
```
10.2.2 
Customized Approach Objective
Sufficient data to be able to identify successful and
failed attempts and who, what, when, where, and
how for each event listed in requirement 10.2.1 are
captured.
```

I think we should collect the request bodies.